src/server.jsx

```diff
@@ -75,6 +76,7 @@ server.disable('x-powered-by');
 server.use(cookieParser());
 server.use(i18nMiddleware.handle(i18n));
 server.use(express.static(DIST_PATH));
+server.use('/', oembed);
```

Comment:
`.use('/oembed')` here, and change the path inside router to '/' seems cleaner and safe?